exo-go/src/cmd/test.go

```diff
@@ -21,6 +23,14 @@ var testCmd = &cobra.Command{
 		if err != nil {
 			panic(err)
 		}
+		serviceName := ""
+		if _, err = os.Stat("service.yml"); err == nil {
+			_, serviceName = path.Split(appDir)
```

Comment:
I don't think the directory name will always match the serviceName. See space-tweet for an example of when they are different.